### ISSUE TITLE
Interactive exploration

### ISSUE THREAD

**Issue by viebel** [stathissideris/spec-provider#6]
Great tool!

Would you like to write an interactive blog post in order to help people experiment with this tool?
You could use Klipse like I did with [this article about clojure.spec](http://blog.klipse.tech/clojure/2016/10/10/defn-args.html)...

**Comment by stathissideris** [comment]
Glad you like it! Interactive blog post sounds like a good idea, but would I need to cross-compile spec-provider to clojurescript first?

**Comment by viebel** [comment]
Yeah, there is a guide available [here](https://github.com/viebel/klipse/wiki/How-to-make-a-clojure-library-self-host-compatible).

Basically, you need to:
1. renaming your files from clj to cljc
2. moving the macros into a separate clj file

**Comment by stathissideris** [comment]
Yeah, I know what it involves 😄  luckily spec-provider is interop-free, so it should be easy,

**Comment by stathissideris** [comment]
This should now be possible! I just need to find some time to do it.
